src/challengePaths/2015/CommunityService.ts

```diff
@@ -129,20 +129,29 @@ class Test {
    * @param beCertain Whether we should check council.php instead of mafia to determine whether the test is complete.
    * @returns The output of the prepare function given, or null if the test is already complete.
    */
-  run<T>(prepare: () => T, beCertain = false): T | null {
+  run<T>(
+    prepare: () => T,
+    beCertain = false,
+    maxTurns: number | null = null
```

Comment:
if you want to be wanky you can do `maxTurns = Infinity` here and save some logic further down

Reply:
I do want to be wanky. I was considering this, and now I'll go ahead and pull the trigger.